### ISSUE TITLE
Don't output dummy process rows?

### ISSUE THREAD

**Issue by siddharth-krishna** [etsap-TIMES/xl2times#174]
> > I tried to implement (a), but on Demo 4 that still leaves us with 3 additional rows (the original ACT_COST = 1111 rows).
>
> I am seeing them **hard-coded** in your code, `def generate_dummy_processes`. :grin:   Apparently VEDA2 does not write out such "internal" ACT_COST default values for the dummy imports (VEDA-FE actually did).

_Originally posted by @Antti-L in https://github.com/etsap-TIMES/xl2times/issues/173#issuecomment-1923681082_

**Comment by olejandro** [comment]
Yes, that probably was one of my very first contributions to the codebase. :-)

I can introduce a transform that cleans this up once we use `sourcescen` consistently.

**Comment by olejandro** [comment]
We should probably allow disabling dummy process generation, as well.

**Comment by Antti-L** [comment]
Note also that the values are not consistent with what VEDA2 assumes, and so you could easily have "regressions" if TFM_UPD is used with multipliers.  But I guess it doesn't matter too much...
